### ISSUE TITLE
Integration with Laravel

### ISSUE THREAD

**Issue by bilogic** [soerennb/extplorer#27]
Any known integration efforts with Laravel?

**Comment by soerennb** [comment]
No, there have been idea about it years ago, but I've never worked with Laravel since, so I'm not the right person to develop for this framework.

**Comment by bilogic** [comment]
Thanks, I had a quick look and think yours is most suitable to "modernize". See https://github.com/koreader/koreader/issues/3193

But it has quite a number of legacy syntax such as `{0}` to access an array. Hope you will be open to PR, but it probably means dropping support for PHP 5.x
